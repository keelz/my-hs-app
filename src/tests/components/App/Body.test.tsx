import * as React from 'react';
import * as enzyme from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import AppBody from '../../../components/App/Body';

describe('AppBody', () => {
  describe('snapshots', () => {
    it('renders correctly', () => {
      const tree = enzyme.shallow(<AppBody />);
      expect(shallowToJson(tree)).toMatchSnapshot();
    });
  });
});
