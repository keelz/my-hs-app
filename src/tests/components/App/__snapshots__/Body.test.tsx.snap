// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppBody snapshots renders correctly 1`] = `
<section
  className="App-body"
>
  <AppRouter />
</section>
`;
